MOBILE_API.xml

```diff
@@ -2584,8 +2479,7 @@
             <description>The software version of the system that implements the SmartDeviceLink core.</description>
         </param>
         <param name="iconResumed" type="Boolean" mandatory="false">
-            
-            <description>Existence of apps icon at system. If true, apps icon was resumed at system. If false, apps icon is not resumed at system</description>
+            <description>True: Cached App Icon was used. - No need to set App Icon False: Using of cached App Icon failed. - The App has to set the App Icon.</description>
```

Comment:
`True: Cached App Icon was used; no need to set App Icon. False: Using of cached App Icon failed; the App has to set the App Icon.`